extensions/git/package.nls.json

```diff
@@ -63,6 +63,7 @@
 	"command.showOutput": "Show Git Output",
 	"command.ignore": "Add to .gitignore",
 	"command.revealInExplorer": "Reveal in Side Bar",
+	"command.rebaseAbort": "Rebase Abort",
```

Comment:
Maybe let's change the order, so it's more Englishy

```suggestion
	"command.rebaseAbort": "Abort Rebase",
```

Reply:
Switched the order.